bundler/spec/bundler/friendly_errors_spec.rb

```diff
@@ -5,29 +5,6 @@
 require "cgi"
 
 RSpec.describe Bundler, "friendly errors" do
-  context "with invalid YAML in .gemrc" do
-    before do
-      File.open(home(".gemrc"), "w") do |f|
-        f.write "invalid: yaml: hah"
-      end
-    end
-
-    after do
-      FileUtils.rm(home(".gemrc"))
-    end
-
-    it "reports a relevant friendly error message" do
-      gemfile <<-G
-        source "#{file_uri_for(gem_repo1)}"
-        gem "rack"
-      G
-
-      bundle :install, :env => { "DEBUG" => "true" }
-
-      expect(err).to include("Failed to load #{home(".gemrc")}")
-    end
-  end
```

Comment:
Why was this test removed? What happens now when having an invalid `.gemrc` file?

Reply:
I restored this test and behavior at https://github.com/rubygems/rubygems/pull/6624